doc/user/content/monitoring/_index.md

```diff
@@ -70,6 +70,10 @@ Materialize provides a [recommended dashboard][dashboard-json] that you can [imp
 Grafana][graf-import]. It relies on you having configured prometheus to scrape
 materialized.
 
+## Datadog
+
+Materialize metrics can be sent to Datadog via the [OpenMetrics agent check](https://www.datadoghq.com/blog/monitor-prometheus-metrics/).  (Requires Datadog Agent 6 and above). Simply configure _prometheus_url_ (ie `http://<materialized host>/metrics`), namespace, and metrics (ie `mz*`) in _openmetrics.d/conf.yaml_.
```

Comment:
We don't lint on this so TIOLI but most of these files have a line length of at most 100 characters, and it is slightly nice to keep to that.

Reply:
Also I think both "prometheus_url" and the conf file possibly should be in quotes? I think that there are genuine philosophies around this and I'm not sure that I'm right, your call.

Reply:
makes sense!  i updated based on both of those recommendations